independent-publisher-connectors/WebEx/apiDefinition.swagger.json

```diff
@@ -0,0 +1,291 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "Webex Integration",
+    "description": "WebEx API for Power Platform\n\n- Setup meeting from Power Platform",
+    "contact": {
+      "name": "UCL",
+      "email": "oscar.hui.21@ucl.ac.uk"
+    }
+  },
+  "host": "webexapis.com",
+  "basePath": "/v1",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/meetings": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
```

Comment:
@ocarhui Please change to a proper description. for example you can use "OK", "Success", "Bad Request", ...etc.
Please fix all responses' descriptions in your connector swagger file.